Remotion/Web/ClientScript/Scripts/DropDownMenu.ts

```diff
@@ -1017,52 +1012,52 @@ class DropDownMenu
     }
   }
 
-  private static GetItemEnabled (itemInfo: DropDownMenu_ItemInfo, selectionCount: number): boolean
+  private static GetItemEnabled(itemInfo: DropDownMenu_ItemInfo, selectionCount: number): boolean
   {
-    var isEnabled = true;
+    let isEnabled = true;
     if (itemInfo.IsDisabled)
     {
       isEnabled = false;
     }
     else
     {
       if (itemInfo.RequiredSelection == DropDownMenu._requiredSelectionExactlyOne
-          && selectionCount != 1)
+        && selectionCount != 1)
```

Comment:
this indet should be double-indent
```suggestion
          && selectionCount != 1)
```